site/hxapi/viewamendcancel/view.md

```diff
@@ -99,7 +99,7 @@ For a detailed explanation of the fields returned, please see below:
 | AvailableSupplements/Supplement/ChDiscAmt      | The amount of discount applied to the child price. Only present for per person upgrades                                                                                                                       |
 | AvailableSupplements/Supplement/PerNight       | Whether the upgrade is charged per night of a stay                                                                                                                                                            |
 | API_Header/Request                             | A list of parameters that were sent in the view booking request.                                                                                                                                              |
-
+| Booking/BookingStatus  (Future)                | Indicates the current status of the booking (`PENDING`, `CONFIRMED`, `CANCELLED`, or `FAILED`).                                                                                                               |
```

Comment:
This doesn't entirely match https://github.com/holidayextras/hapi/blob/staging/controllers/legacy/lib/bookingStatus.js#L5

Also, we don't send them as upper case, do we?